docs/development/doc_guide.rst

```diff
@@ -1,76 +1,849 @@
+warnings. A tox_ command may also include the ``--parallel`` flag to
+allow a parallel build.
+
+Documentation tools
+===================
+
+ReStructuredText
+----------------
+
+PlasmaPy's documentation is written using the `reStructuredText (reST)`_
+markup language. reST_ is human readable when viewed within a source
+code file or when printed out using `help`. reST_ also contains markup
+that allows the text to be transformed into `PlasmaPy's documentation`_.
+reST_ files end in :file:`.rst`. Documentation contained within
+:file:`.py` files are in the form of docstrings_, which are written in
+reST_.
+
+ReStructuredText Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here we show some examples of commonly used reST_ syntax in
+PlasmaPy.  Please refer to the documentation for Sphinx_ and reST_ for
+a list of available directives and syntax.
+
+This is an example of including headings for the document title,
+sections, subsections, and so on. The lines surrounding each heading are
+the same length as that heading.
+
+.. code-block:: rst
+
+   ==============
+   Document title
+   ==============
+
+   Heading 1
+   =========
+
+   Heading 2
+   ---------
+
+   Heading 3
+   ~~~~~~~~~
+
+We can link to code objects by enclosing them in single back ticks.
+This linking will work for Python_ commands as well as certain packages
+like NumPy_, SciPy_, Astropy_, and pandas_. This linking is described in
+the section on :ref:`external-references`. In-line code examples are
+enclosed in double back ticks.
+
+.. code-block:: rst
+
+   Here is a reference to `plasmapy.particles` that will show the full
+   namespace in the rendered documentation.
+
+   Only the word "Particle" will show up if a prepend a tilde like in
+   `~plasmapy.particles.particle_class.Particle`.
+
+   Here is some in-line code: ``import astropy.units as u``.
+
+This reST_ block will render as:
+
+   Here is a reference to `plasmapy.particles` that will show the full
+   namespace in the rendered documentation.
+
+   Only the word "Particle" will show up if a prepend a tilde like in
+   `~plasmapy.particles.particle_class.Particle`.
+
+   Here is some in-line code: ``import astropy.units as u``.
+
+Sphinx_ can format code blocks for Python_ and the Python_ console.
+
+   .. code-block:: rst
+
+      .. code-block:: python
+
+         def sample_function():
+             return 42
+
+      .. code-block:: pycon
+
+         >>> print(6 * 9)
+         54
+
+This reST block will render as:
+
+   .. code-block:: python
+
+      def sample_function():
+          return 42
+
+   .. code-block:: pycon
+
+      >>> print(6 * 9)
+      54
+
+Here are some examples for linking to websites.
+
+.. code-block:: rst
+
+   Here is a link to `PlasmaPy's website <https://www.plasmapy.org>`_.
+
+   We can link to PlasmaPy's latest documentation_ or `Python's website`_.
+
+   .. _documentation: https://docs.plasmapy.org/en/latest/
+   .. _`Python's website`: https://www.python.org/
```

Comment:
Is there a reason you did not include a rendered version here?

Reply:
Yes, it was to avoid duplication with `common_links.rst`, at least for earlier commits.  With that said, yeah, it'd be worth changing the example so we can include a rendered version.